src/module.ts

```diff
@@ -1477,6 +1485,7 @@ export class Module {
     this.setOptimizeLevel(optimizeLevel);
     this.setShrinkLevel(shrinkLevel);
     this.setDebugInfo(debugInfo);
+    this.setFastMath(true);
```

Comment:
Compare with clang's `ffast-math` It's much safer and just don't care about NaN bits consistency for non-canonical NaNs and accept only safe optimizations like `x * 1.0` -> `x`, `x / 1.0` -> `x`, `x * -1.0` -> `-x`, `x - 0.0` -> `x`, and still doesn't allow unsafe ones like `x + 0.0` -> `x`, `x - x` -> `0` and etc.

So always turn it on by default.